public/_general.scss

```diff
@@ -90,36 +100,60 @@ a {
   height: 90px;
 }
 
+#main #hero {
+  color: white;
```

Comment:
Somehow on small screens, the changes cause the background to be too small for the hero text
![Bildschirmfoto 2019-08-12 um 18 00 17](https://user-images.githubusercontent.com/2821879/62879326-28f68a00-bd2b-11e9-9d48-381555c93b42.png)

Reply:
tried to solve this pragmatically. Pushed a fix

Reply:
Still a problem on other pages like /tickets